index.js

```diff
@@ -450,12 +449,14 @@ function runDataIntegrityProofVerifyTests({
       const credential = credentials.clone('invalidCreated');
       await verificationFail({credential, verifier});
     });
-    it('If the "proof.proofValue" field is not a multibase-encoded ' +
-      'base58-btc value, an error MUST be raised.', async function() {
+    it('If the "proof.proofValue" field is not multibase-encoded, an error ' +
+      'MUST be raised.', async function() {
       this.test.cell = {columnId, rowId: this.test.title};
       const credential = credentials.clone('issuedVc');
-      // remove the initial z
+
+      // Remove the multibase header
```

Comment:
```suggestion
      // Remove the multibase header to cause validation error
```

Probably best to explain "why" as well.